src/merkleblock.h

```diff
@@ -128,6 +128,9 @@ class CMerkleBlock
     /** Public only for unit testing */
     CBlockHeader header;
     CPartialMerkleTree txn;
+    bool m_prove_gentx;
```

Comment:
3abfd6d3d99bbcceb09162e83d8f9f9769bd4036: what does this do?

Reply:
An attempt to have a somewhat optional extension, but given the need to prove wtxid for non-witness txs, maybe not the best approach.